### DISCUSSION TITLE
[Help]: Uncaught TypeError: Cannot read properties of undefined (reading 'map')

### DISCUSSION THREAD

**Discussion by janstieler** [davidjerleke/embla-carousel#972] [Help] [answered]
### Summary

I got the following error:
```js
embla-carousel-auto-height.umd.js:1 Uncaught TypeError: Cannot read properties of undefined (reading 'map')
    at embla-carousel-auto-height.umd.js:1:442
    at i (embla-carousel-auto-height.umd.js:1:490)
    at Object.init (embla-carousel-auto-height.umd.js:1:668)
    at embla-carousel.umd.js:2:18768
    at Array.forEach (<anonymous>)
    at Object.init (embla-carousel.umd.js:2:18752)
    at A (embla-carousel.umd.js:2:20228)
    at H (embla-carousel.umd.js:2:21701)
    at referenzen:202:34
    at NodeList.forEach (<anonymous>)
```
if I use the autoheight plugin. 
On my [test](https://codepen.io/janstieler-the-bashful/pen/xxoXzZE) codepen I havn't this error. But I can't really understand whats the problem is on my running site.

Eventuell someone can say me whats happend?

Best

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [X] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [X] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### CodeSandbox example

https://codepen.io/janstieler-the-bashful/pen/xxoXzZE

**Comment by davidjerleke** [comment, marked as answer]
Hi @janstieler,

Thanks for your question. This was also reported here:

- #1009

And fixed in this **pull request**:

- #1012

The fix will be released with the next version `v8.3.1`.
